### DISCUSSION TITLE
Need Help to solve error in vscode setup

### DISCUSSION THREAD

**Discussion by Deepanshuchawla** [microsoft/vscode-discussions#649] [Extension Development QnA] [answered]
Hi I need help with setup in vs code 
In my vs code, I can run the code easily but if I want to debug my C++ code it will show me this error "the prelaunch task 'c/c++ g++ build active file' terminated with exit code 1 "   
Can anyone help me to solve this problem?

**Comment by gjsjohnmurray** [comment, marked as answer]
This forum focuses on developers who are creating new extensions for VS Code. It doesn't sound like that's what you are doing.
